### PR TITLE
Fix token command for non default ports

### PR DESCRIPTION
**Pull request by seanh**
Also improve the docs

### CODE REVIEW

**Comment by seanh** [comment]
@nickstenning

**Comment by seanh** [comment]
Wondering about adding a mention of this command to our API docs in api.rst as well. It's only useful to developers working on their local instance as far as I can see (you have to pass the paster config file to the command), but at least those devs will be able to find it

**Comment by tilgovi** [comment]
Is the indentation on that docstring strange?

File: h/script.py [hunk 1]
```diff
@@ -118,12 +133,18 @@ def token(args):
 
     print(token)
 
-parser_token = subparsers.add_parser('token', help=token.__doc__)
+parser_token = subparsers.add_parser(
+    'token', description=token.__doc__,
```
Comment (nickstenning) [inline h/script.py]: If you could employ `textwrap.dedent` here ([c.f. `hypothesis-buildext`](https://github.com/hypothesis/h/blob/9f9496198e8177653dba5781fcbefed57f878be8/h/buildext.py#L267)) and reformat the docstring to look a little more conventional, that would make me happy :).

File: h/script.py [hunk 2]
```diff
@@ -98,15 +103,25 @@ def reindex(args):
...
-    """Generate an OAuth bearer token for the specified principal."""
+    """Generate an OAuth bearer token for the specified principal.
+
+This token is suitable for passing to the h API as the value of an
+X-Annotator-Auth-Token header.
+
+For example, to authorize yourself as user seanh to your local dev instance
+of h do:
+
+    hypothesis token --base 'http://localhost:5000' --sub 'acct:seanh@127.0.0.1' conf/development.ini
```
Comment (nickstenning) [inline h/script.py]: Also this should probably read "localhost" not "127.0.0.1".

**Pull request merged**